Update nginx-ingress-controller to version 0.30.0

Ebaneck opened the pull request:
**Component**:

<!-- E.g. 'salt', 'containers', 'kubernetes', 'build', 'tests'... -->
'charts', 'nginx-ingress'

**Context**: 

See #2423 

**Summary**:

Bump the Nginx-ingress charts to most recent stable release

**Acceptance criteria**: 

No regression

```2020-04-22T11:31:01.353066567Z stdout F -------------------------------------------------------------------------------
2020-04-22T11:31:01.35310544Z stdout F NGINX Ingress controller
2020-04-22T11:31:01.353110594Z stdout F   Release:       0.30.0
2020-04-22T11:31:01.353113455Z stdout F   Build:         git-7e65b90c4
2020-04-22T11:31:01.353116684Z stdout F   Repository:    https://github.com/kubernetes/ingress-nginx
2020-04-22T11:31:01.353119765Z stdout F   nginx version: nginx/1.17.8
2020-04-22T11:31:01.353122707Z stdout F 
2020-04-22T11:31:01.353125829Z stdout F -------------------------------------------------------------------------------


```


---

<!-- Declare one or more issues to close once this PR gets merged -->

Closes: #2423 

<!-- If you want to refer to an issue while not closing it, use:

See: #ISSUE_NUMBER

-->

Ebaneck committed:
charts: Update nginx-ingress to version 0.30.0

This chart is updated using:

```
$ rm -rf charts/nginx-ingress/
$ helm fetch -d charts --untar stable/nginx-ingress
$ ./charts/render.py nginx-ingress --namespace metalk8s-ingress charts/nginx-ingress.yaml charts/nginx-ingress/ > salt/metalk8s/addons/nginx-ingress/deployed/chart.sls
```

bert-e commented:
# Hello ebaneck,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://eve.devsca.com/github/scality/metalk8s/bert-e/doc/user).





*Status report is not available.*

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `improvement/2423-upgrade-nginx-ingress-controller-to-0.30.0` into
`development/2.5`
* [w/2.6/improvement/2423-upgrade-nginx-ingress-controller-to-0.30.0](https://github.com/scality/metalk8s/compare/development/2.6...w/2.6/improvement/2423-upgrade-nginx-ingress-controller-to-0.30.0) will be merged into `development/2.6` 



The following branches will **NOT** be impacted:

* `development/1.0`
* `development/1.1`
* `development/1.2`
* `development/1.3`
* `development/2.0`
* `development/2.1`
* `development/2.2`
* `development/2.3`
* `development/2.4`




You can set option `create_pull_requests` if you need me to create
**integration pull requests** in addition to integration branches, with:

```
@bert-e create_pull_requests
```

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @nootal

* @alexandre-allard-scality

* @lucieleonard

* @sayf-eddine-scality

* @slaperche-scality

* @nicolast

* @gdemonet

* @carlito-scality

* @thomasdanan

* @jbertran

* @teddyandrieux

* @cmonfort

* @monpote

Ebaneck commented:
/after_pull_request 2441

Ebaneck commented:
/approve

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @nootal

* @alexandre-allard-scality

* @lucieleonard

* @sayf-eddine-scality

* @slaperche-scality

* @nicolast

* @gdemonet

* @carlito-scality

* @thomasdanan

* @jbertran

* @teddyandrieux

* @cmonfort

* @monpote










*The following options are set:* **approve**

bert-e commented:
# History mismatch



Merge commit #aa8e5b8749e659c3fb01271b92087e4dc515e418 on the integration branch
`w/2.6/improvement/2423-upgrade-nginx-ingress-controller-to-0.30.0` is merging a branch which is neither the current
branch `improvement/2423-upgrade-nginx-ingress-controller-to-0.30.0` nor the development branch
`development/2.6`.

It is likely due to a rebase of the branch `improvement/2423-upgrade-nginx-ingress-controller-to-0.30.0` and the
merge is not possible until all related `w/*` branches are deleted or updated.

**Please use the `reset` command to have me reinitialize these branches.**





*The following options are set:* **approve**

Ebaneck commented:
/reset

Ebaneck commented:
/approve

bert-e commented:
# Reset complete



I have successfully deleted this pull request's integration branches.






*The following options are set:* **approve**

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `improvement/2423-upgrade-nginx-ingress-controller-to-0.30.0` into
`development/2.5`
* [w/2.6/improvement/2423-upgrade-nginx-ingress-controller-to-0.30.0](https://github.com/scality/metalk8s/compare/development/2.6...w/2.6/improvement/2423-upgrade-nginx-ingress-controller-to-0.30.0) will be merged into `development/2.6` 



The following branches will **NOT** be impacted:

* `development/1.0`
* `development/1.1`
* `development/1.2`
* `development/1.3`
* `development/2.0`
* `development/2.1`
* `development/2.2`
* `development/2.3`
* `development/2.4`




You can set option `create_pull_requests` if you need me to create
**integration pull requests** in addition to integration branches, with:

```
@bert-e create_pull_requests
```







*The following options are set:* **approve**

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @nootal

* @alexandre-allard-scality

* @lucieleonard

* @sayf-eddine-scality

* @slaperche-scality

* @nicolast

* @gdemonet

* @carlito-scality

* @thomasdanan

* @jbertran

* @teddyandrieux

* @cmonfort

* @monpote










*The following options are set:* **approve**

bert-e commented:
# History mismatch



Merge commit #311142c648531c3a4787780998a6bdd23f15d724 on the integration branch
`w/2.6/improvement/2423-upgrade-nginx-ingress-controller-to-0.30.0` is merging a branch which is neither the current
branch `improvement/2423-upgrade-nginx-ingress-controller-to-0.30.0` nor the development branch
`development/2.6`.

It is likely due to a rebase of the branch `improvement/2423-upgrade-nginx-ingress-controller-to-0.30.0` and the
merge is not possible until all related `w/*` branches are deleted or updated.

**Please use the `reset` command to have me reinitialize these branches.**





*The following options are set:* **approve**

bert-e commented:
# History mismatch



Merge commit #311142c648531c3a4787780998a6bdd23f15d724 on the integration branch
`w/2.6/improvement/2423-upgrade-nginx-ingress-controller-to-0.30.0` is merging a branch which is neither the current
branch `improvement/2423-upgrade-nginx-ingress-controller-to-0.30.0` nor the development branch
`development/2.6`.

It is likely due to a rebase of the branch `improvement/2423-upgrade-nginx-ingress-controller-to-0.30.0` and the
merge is not possible until all related `w/*` branches are deleted or updated.

**Please use the `reset` command to have me reinitialize these branches.**





*The following options are set:* **approve**

Ebaneck commented:
/reset

bert-e commented:
# Reset complete



I have successfully deleted this pull request's integration branches.






*The following options are set:* **approve**

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `improvement/2423-upgrade-nginx-ingress-controller-to-0.30.0` into
`development/2.5`
* [w/2.6/improvement/2423-upgrade-nginx-ingress-controller-to-0.30.0](https://github.com/scality/metalk8s/compare/development/2.6...w/2.6/improvement/2423-upgrade-nginx-ingress-controller-to-0.30.0) will be merged into `development/2.6` 



The following branches will **NOT** be impacted:

* `development/1.0`
* `development/1.1`
* `development/1.2`
* `development/1.3`
* `development/2.0`
* `development/2.1`
* `development/2.2`
* `development/2.3`
* `development/2.4`




You can set option `create_pull_requests` if you need me to create
**integration pull requests** in addition to integration branches, with:

```
@bert-e create_pull_requests
```







*The following options are set:* **approve**

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @nootal

* @alexandre-allard-scality

* @lucieleonard

* @sayf-eddine-scality

* @slaperche-scality

* @nicolast

* @gdemonet

* @carlito-scality

* @thomasdanan

* @jbertran

* @teddyandrieux

* @cmonfort

* @monpote










*The following options are set:* **approve**

Ebaneck committed:
charts: Update nginx-ingress-control-plane to version 0.30.0

This chart is updated using:

```
$ rm -rf charts/nginx-ingress/
$ helm fetch -d charts --untar stable/nginx-ingress
$ ./charts/render.py nginx-ingress-control-plane --namespace metalk8s-ingress charts/nginx-ingress-control-plane.yaml
charts/nginx-ingress/ > salt/metalk8s/addons/nginx-ingress-control-plane/deployed/chart.sls
```

bert-e commented:
# History mismatch



Merge commit #898bb23a21efece61e206d145aa16a1ba289f84d on the integration branch
`w/2.6/improvement/2423-upgrade-nginx-ingress-controller-to-0.30.0` is merging a branch which is neither the current
branch `improvement/2423-upgrade-nginx-ingress-controller-to-0.30.0` nor the development branch
`development/2.6`.

It is likely due to a rebase of the branch `improvement/2423-upgrade-nginx-ingress-controller-to-0.30.0` and the
merge is not possible until all related `w/*` branches are deleted or updated.

**Please use the `reset` command to have me reinitialize these branches.**





*The following options are set:* **approve**

Ebaneck commented:
/reset

bert-e commented:
# Reset complete



I have successfully deleted this pull request's integration branches.






*The following options are set:* **approve**

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `improvement/2423-upgrade-nginx-ingress-controller-to-0.30.0` into
`development/2.5`
* [w/2.6/improvement/2423-upgrade-nginx-ingress-controller-to-0.30.0](https://github.com/scality/metalk8s/compare/development/2.6...w/2.6/improvement/2423-upgrade-nginx-ingress-controller-to-0.30.0) will be merged into `development/2.6` 



The following branches will **NOT** be impacted:

* `development/1.0`
* `development/1.1`
* `development/1.2`
* `development/1.3`
* `development/2.0`
* `development/2.1`
* `development/2.2`
* `development/2.3`
* `development/2.4`




You can set option `create_pull_requests` if you need me to create
**integration pull requests** in addition to integration branches, with:

```
@bert-e create_pull_requests
```







*The following options are set:* **approve**

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @nootal

* @alexandre-allard-scality

* @lucieleonard

* @sayf-eddine-scality

* @slaperche-scality

* @nicolast

* @gdemonet

* @carlito-scality

* @thomasdanan

* @jbertran

* @teddyandrieux

* @cmonfort

* @monpote










*The following options are set:* **approve**

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/2.5`

* :heavy_check_mark: `development/2.6`



The following branches will **NOT** be impacted:

* `development/1.0`
* `development/1.1`
* `development/1.2`
* `development/1.3`
* `development/2.0`
* `development/2.1`
* `development/2.2`
* `development/2.3`
* `development/2.4`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/2.5`

* :heavy_check_mark: `development/2.6`



The following branches have **NOT** changed:

* `development/1.0`
* `development/1.1`
* `development/1.2`
* `development/1.3`
* `development/2.0`
* `development/2.1`
* `development/2.2`
* `development/2.3`
* `development/2.4`



Please check the status of the associated issue None.

Goodbye ebaneck.
